src/modules/roc_fec/writer.cpp

```diff
@@ -68,51 +69,85 @@ void Writer::write(const packet::PacketPtr& pp) {
         } while (source_ == pp->rtp()->source);
     }
 
+    if (cur_packet_ == 0) {
+        if (!begin_block_()) {
+            return;
```

Comment:
As discussed, we should either set alive_ to false here or better skip current block.

Reply:
As discussed the panic was added :)

Reply:
Will be fixed in #179.